2025-10-24 11:46:13,840 : INFO : root : Loading config.
2025-10-24 11:46:13,844 : INFO : YamlMultifileConfigLoader : Machine config file #1: config.yaml
2025-10-24 11:46:13,844 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\fdca14d0c9d658ab8200d57120800af5.mpf_cache
2025-10-24 11:46:13,992 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\5dec96b1547c4adf5bdd3d1715e74bc0.mpf_cache
2025-10-24 11:46:13,993 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\202a5840610e95051f9f473dfa660cc0.mpf_cache
2025-10-24 11:46:13,995 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\5dec96b1547c4adf5bdd3d1715e74bc0.mpf_cache
2025-10-24 11:46:13,995 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\2cd13a2d3f375e00ab01d6d09a85d8e7.mpf_cache
2025-10-24 11:46:13,996 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\9220b44b97b91972524500b81035d790.mpf_cache
2025-10-24 11:46:13,998 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\dab29d17bbe6500d014c71b0a57bd9b0.mpf_cache
2025-10-24 11:46:13,998 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\b17152efa55a24f0424602723f8d3921.mpf_cache
2025-10-24 11:46:13,999 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\e87fc0e7d2f8a851b5fa3f604e0d0c6e.mpf_cache
2025-10-24 11:46:14,000 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\196ac76b6cab3a9880804c826cfbcb5c.mpf_cache
2025-10-24 11:46:14,000 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\834292dd7853e12a66732dc89828c828.mpf_cache
2025-10-24 11:46:14,001 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\ae2be9c5579852c305357e207df97e8b.mpf_cache
2025-10-24 11:46:14,002 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\46f8fa3e41dfeeb8232af2f03a13b6c4.mpf_cache
2025-10-24 11:46:14,003 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\c67e9c175da6c060302945929b27b878.mpf_cache
2025-10-24 11:46:14,004 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\1c965aadbb61a7e709e8884fad17ac0b.mpf_cache
2025-10-24 11:46:14,005 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\60ca9d662df3e70e96f6d7ec0f4491aa.mpf_cache
2025-10-24 11:46:14,006 : INFO : Machine : Mission Pinball Framework Core Engine v0.80.0.dev11
2025-10-24 11:46:14,006 : INFO : Machine : Command line arguments: {'no_load_cache': False, 'create_config_cache': True, 'bcp': False, 'configfile': ['config.yaml'], 'force_assets_load': False, 'platform_integration_test': False, 'jsonlogging': False, 'logfile': 'logs\\2025-10-24-11-46-13-mpf-LAPTOP-MPGP6R69.log', 'pause': False, 'production': False, 'text_ui': False, 'loglevel': 15, 'consoleloglevel': 20, 'force_platform': 'virtual', 'syslog_address': None, 'full_logfile_path': 'C:\\Users\\antho\\nightpin\\logs\\2025-10-24-11-46-13-mpf-LAPTOP-MPGP6R69.log'}
2025-10-24 11:46:14,006 : INFO : Machine : MPF path: C:\Users\antho\mpf\Lib\site-packages\mpf
2025-10-24 11:46:14,006 : INFO : Machine : Machine path: C:\Users\antho\nightpin
2025-10-24 11:46:14,006 : INFO : Machine : Platform: win32
2025-10-24 11:46:14,006 : INFO : Machine : Python executable location: C:\Users\antho\mpf\Scripts\python.exe
2025-10-24 11:46:14,006 : INFO : Machine : Python version: 3.12.7 (64-bit)
2025-10-24 11:46:14,009 : INFO : Machine : Initializing MPF...
2025-10-24 11:46:14,009 : ERROR : ConfigValidator : Your config contains a value for the setting "hardware:default_pulse_ms", but this is not a valid setting name.
2025-10-24 11:46:14,010 : ERROR : Machine : Failed to initialize MPF
Traceback (most recent call last):
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\machine.py", line 688, in initialize_mpf
    raise init.exception()
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\machine.py", line 250, in initialize
    await self.initialize_core_and_hardware()
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\machine.py", line 236, in initialize_core_and_hardware
    self._load_hardware_platforms()
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\machine.py", line 475, in _load_hardware_platforms
    self.validate_machine_config_section('hardware')
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\machine.py", line 393, in validate_machine_config_section
    self.config[section] = self.config_validator.validate_config(
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_validator.py", line 164, in validate_config
    return self._validate_config(config_spec, source, base_spec=base_spec, add_missing_keys=add_missing_keys,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_validator.py", line 177, in _validate_config
    self.check_for_invalid_sections(this_spec, source, validation_failure_info)
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_validator.py", line 345, in check_for_invalid_sections
    raise ConfigFileError('Your config contains a value for the '
mpf.exceptions.config_file_error.ConfigFileError: Config File Error in ConfigValidator: Your config contains a value for the setting "hardware:default_pulse_ms", but this is not a valid setting name. Error Code: CFE-ConfigValidator-2 (https://missionpinball.org/logs)
2025-10-24 11:46:14,013 : INFO : root : MPF run loop ended.
